node/bin/src/prover_api/snark_job_manager.rs

```diff
@@ -243,43 +164,57 @@ impl SnarkJobManager {
         Ok(())
     }
 
-    /// Consumes fake FRI proves from HEAD and turns them into fake SNARKs
-    /// Additionally, if `consume_by_timeout` is Some,
-    ///    also consumes real FRI proves that are older than `consume_by_timeout`
-    async fn consume_fake_proves_from_head(
+    /// Consumes fake FRI proofs from the head of the queue and turns them into fake SNARKs.
+    async fn process_pending_fake_fri_proofs(&self) -> anyhow::Result<()> {
```

Comment:
Will this work as intended? We call it in front of the `pick_real_jobs` which means it will consume all the fake proofs (as intended) but also timed out proofs (those would be the ones we sent and didn't receive an answer for, right?).

Reply:
No, the comment is enough. The order of reading messed me up. My miss, don't mind me.